incubator/gocd/templates/gocd-server-godata-pvc.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if and .Values.server.persistence.enabled (not .Values.server.persistence.godata.existingClaim) -}}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ .Values.server.persistence.godata.name }}
+  labels:
+    app: {{ template "gocd.name" . }}-server
+    chart: "{{ template "gocd.name" . }}-{{ .Chart.Version }}"
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
+    role: server
+spec:
+  accessModes:
+    - {{ .Values.server.persistence.godata.accessMode | quote }}
+  resources:
+    requests:
+      storage: {{ .Values.server.persistence.godata.size | quote }}
+{{- if .Values.server.persistence.godata.storageClass }}
+  storageClassName: {{ .Values.server.persistence.godata.storageClass }}
+{{- else }}
+  storageClassName: ""
+  selector:
```

Comment:
Selectors not needed.

Reply:
@unguiculus -- GoCD server needs 2 Volumes, one to store `godata` and the other one for `homego`. Without selectors, is there any way of binding of a PVC to a specific persistent volume?

Reply:
There is not a single chart that uses selectors on PVCs. However, they may be necessary when you want to specify an existing PV. Otherwise, when the PV is auto-provisioned you should be fine without selectors. If you do want to allow selectors, then do it like so:

```yaml
{{- if .Values.server.persistence.godata.pvSelector }}
  selector: 
{{- toYaml .Values.server.persistence.godata.pvSelector | indent 4 }}
{{- end }}
```

Apply this for all PVCs.